src/cmds/plots.py

```diff
@@ -173,30 +28,139 @@ def show(root_path):
+@click.option("-b", "--buffer", help="Megabytes for sort/plot buffer", type=int, default=4608, show_default=True)
+@click.option("-r", "--num_threads", help="Number of threads to use", type=int, default=2, show_default=True)
+@click.option("-u", "--buckets", help="Number of buckets", type=int, default=0)
+@click.option("-s", "--stripe_size", help="Stripe size", type=int, default=0)
+@click.option(
+    "-a",
+    "--alt_fingerprint",
+    type=int,
+    default=None,
+    help="Enter the alternative fingerprint of the key you want to use"
+)
+@click.option(
+    "-c",
+    "--pool_contract_address",
+    type=str,
+    default=None,
+    help="Address of where the pool reward will be sent to. Only used if alt_fingerprint and pool public key are None"
+)
+@click.option("-f", "--farmer_public_key", help="Hex farmer public key", type=str, default=None)
+@click.option("-p", "--pool_public_key", help="Hex public key of pool", type=str, default=None)
+@click.option(
+    "-t",
+    "--tmp_dir",
+    help="Temporary directory for plotting files",
+    type=click.Path(),
+    default=Path("."),
+    show_default=True
+)
+@click.option("-2", "--tmp2_dir", help="Second temporary directory for plotting files", type=click.Path(), default=None)
+@click.option(
+    "-d",
+    "--final_dir",
+    help="Final directory for plots (relative or absolute)",
+    type=click.Path(),
+    default=Path("."),
+    show_default=True
+)
+@click.option("-i", "--plotid", help="PlotID in hex for reproducing plots (debugging only)", type=str, default=None)
+@click.option("-m", "--memo", help="Memo in hex for reproducing plots (debugging only)", type=str, default=None)
+@click.option("-e", "--nobitfield", help="Disable bitfield", default=False, is_flag=True)
+@click.option(
+    "-x",
+    "--exclude_final_dir",
+    help="Skips adding [final dir] to harvester for farming",
+    default=False,
+    is_flag=True
+)
+@click.pass_context
+def create_cmd(ctx: click.Context, size: int, num: int, buffer: int, num_threads: int, buckets: int, stripe_size: int,
+               alt_fingerprint: int, pool_contract_address: str, farmer_public_key: str, pool_public_key: str,
+               tmp_dir: str, tmp2_dir: str, final_dir: str, plotid: str, memo: str, nobitfield: bool,
+               exclude_final_dir: bool):
+    class Params(object):
+        size = size
+        num = num
+        buffer = buffer
+        num_threads = num_threads
+        buckets = buckets
+        stripe_size = stripe_size
+        alt_fingerprint = alt_fingerprint
+        pool_contract_address = pool_contract_address
+        farmer_public_key = farmer_public_key
+        pool_public_key = pool_public_key
+        tmp_dir = Path(tmp_dir)
+        tmp2_dir = Path(tmp2_dir)
+        final_dir = Path(final_dir)
+        plotid = plotid
+        memo = memo
+        nobitfield = nobitfield
+        exclude_final_dir = exclude_final_dir
+    create_plots(Params(), ctx.obj["root_path"])
```

Comment:
This function is only explicitly called from here, so, maybe we can move it back to this file, like the rest of the CLI functions. (I don't know if there are other python code importing and using that function from the outside).

Reply:
create_plots is also called from block_tools.py